nautobot/core/templates/generic/object_detail.html

```diff
@@ -0,0 +1,105 @@
+                    {{ verbose_name_plural|bettertitle }}
+                </a></li>
+                {% block extra_breadcrumb %}{% endblock extra_breadcrumb %}
+                {% if object.name %}
+                    <li><a href="{{ object.get_absolute_url }}">{{ object.name }}</a></li>
+                {% else %}
+                    <li><a href="{{ object.get_absolute_url }}">{{ object }}</a></li>
+                {% endif %}
+            {% endblock breadcrumb %}
+            </ol>
+        </div>
+        <div class="col-sm-4 col-md-3">
+            <form action="{% url list_url %}" method="get">
+                <div class="input-group">
+                    <input type="text" name="q" class="form-control" placeholder="Search {{ verbose_name_plural }}" />
+                    <span class="input-group-btn">
+                        <button type="submit" class="btn btn-primary">
+                            <span class="mdi mdi-magnify" aria-hidden="true"></span>
+                        </button>
+                    </span>
+                </div>
+            </form>
+        </div>
+        {% endwith %}
+    </div>
+
+    <div class="pull-right noprint">
+    {% block buttons %}
+        {% block extra_buttons %}{% endblock extra_buttons %}
+        {% plugin_buttons object %}
+        {% if object.clone_fields and user|can_add:object %}
+            {% clone_button object %}
+        {% endif %}
+        {% if user|can_change:object %}
+            {% edit_button object %}
+        {% endif %}
+        {% if user|can_delete:object %}
+            {% delete_button object %}
+        {% endif %}
+    {% endblock buttons %}
+    </div>
+
+    {% block masthead %}
+        <h1>{% block title %}{{ object }}{% endblock %}</h1>
+    {% endblock masthead %}
+    {% include 'inc/created_updated.html' %}
+    {% block panel_buttons %}{% endblock panel_buttons %}
+    <div class="pull-right noprint">
+        {% custom_links object %}
+    </div>
+
+    {% block nav_tabs %}
+    <ul class="nav nav-tabs">
+        <li role="presentation"{% if not active_tab %} class="active"{% endif %}>
+            <a href="{{ object.get_absolute_url }}">
+                {{ verbose_name|bettertitle }}
+            </a>
+        </li>
+        {% block extra_nav_tabs %}{% endblock extra_nav_tabs %}
+        {% if perms.extras.view_objectchange %}
+            <li role="presentation"{% if active_tab == 'changelog' %} class="active"{% endif %}>
+                <a href="{{ changelog_url }}">Change Log</a>
+            </li>
+        {% endif %}
+    </ul>
+    {% endblock nav_tabs %}
+{% endblock header %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-6">
+            {% block content_left_page %}{% endblock content_left_page %}
+            {% include 'inc/custom_fields_panel.html' %}
+            {% include 'inc/relationships_panel.html' %}
+            {% include 'extras/inc/tags_panel.html' %}
```

Comment:
Random thought - so both `custom_fields_panel` (including computed fields as well) and `relationships_panel` can grow arbitrarily long based on the amount of fields/relationships defined. To keep the UI more "balanced", I wonder if we should consider moving `relationships_panel` to the right column instead of keeping all of these in the left column?

Reply:
Yeah I thought about that, as well. I don't have a strong opinion. I was thinking about the "what if" of any of these extras going on the right panel across the board? Splitting them might still result in long columns depending on the object...

Reply:
Can defer this change to a future revision, I think.